Assets/Scripts/Game/GameManager.cs

```diff
@@ -465,7 +465,10 @@ void Update()
             }
             else
                 classicUpdate = false;
+        }
 
+        void Update()
+        {
```

Comment:
Just FYI this change breaks window popping behaviour by allowing pause menu to display when not intended. I didn't catch this one while testing. It's not a big deal though, I'll send a fix shortly. :)

Reply:
Thanks